draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1191,7 +1192,7 @@ Error Handling:
 : If a system overruns the generator by requesting too many UUIDs
   within a single system time interval, the UUID service MUST either
   return an error, or stall the UUID generator until the system clock
-  catches up.
+  catches up, and MUST NOT return knowingly duplicate values.
```

Comment:
I think this communicate the point - an implementation, wherever possible, should prevent a duplicate value from being returned as part of being a correct implementation.  Or to state it another way: if you could have easily prevented returning a value but did not, that's a bug that should be fixed.